tests/src/test/scala/cats/tests/AsTests.scala

```diff
@@ -0,0 +1,24 @@
+package cats
+package tests
+
+class AsTests extends CatsSuite {
+  import evidence._
+
```

Comment:
Can we test resolution explicitly of a few types? `Int As Any` `(Int, Long) As (Any, Any)`